core/remotes/docker/fetcher.go

```diff
@@ -274,11 +297,14 @@ func (r dockerFetcher) open(ctx context.Context, req *request, mediatype string,
 		// Note: "Accept-Ranges: bytes" cannot be trusted as some endpoints
 		// will return the header without supporting the range. The content
 		// range must always be checked.
-		req.header.Set("Range", fmt.Sprintf("bytes=%d-", offset))
+		req.header.Set("range", fmt.Sprintf("bytes=%d-", offset))
```

Comment:
Why'd you change the case of the HTTP headers (in a few places)?

Reply:
Will make everything 'canonical' to avoid operations and for consistency.

https://github.com/golang/go/blob/d36353499f673c89a267a489beb80133a14a75f9/src/net/textproto/reader.go#L647-L666